### PR TITLE
Change affiliation of Madhav Marathe, Anil Vullikanti and Tom Fletcher

### PR DESCRIPTION
**Pull request by jiahao42**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -7553,9 +7553,8 @@ Maarten van Steen,VU Amsterdam,http://www.distributed-systems.net,1mFRcc0AAAAJ
 Maciej Piróg,University of Wroclaw,https://www.cs.ox.ac.uk/people/maciej.pirog,xxwcLlAAAAAJ
 Madalina Fiterau,University of Massachusetts Amherst,http://www.cs.cmu.edu/~mfiterau,NTHsaUQAAAAJ
 Madeleine Udell,Cornell University,https://people.orie.cornell.edu/mru8,tZ9pEDMAAAAJ
-Madhav Marathe,Virginia Tech,https://www.bi.vt.edu/faculty/Madhav-Marathe,cLjMQqsAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:7556]: You *must* change both aliases.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (jiahao42) [reply csrankings.csv:7556]: You mean to update this line as well? I thought it is a duplicate.

Comment (emeryberger) [reply csrankings.csv:7556]: [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in dblp-aliases.csv, add it.

Comment (jiahao42) [reply csrankings.csv:7556]: Sorry for not doing this, now it should be fixed.

Comment (jiahao42) [reply csrankings.csv:7556]: Just realized that other two people's entry need to be fixed as well, and it's done.

File: csrankings.csv [hunk 2]
```diff
@@ -913,7 +913,7 @@ Anil K. Jain,Michigan State University,https://www.cse.msu.edu/~jain,g-_ZXGsAAAA
 Anil K. Jain 0001,Michigan State University,https://www.cse.msu.edu/~jain,g-_ZXGsAAAAJ
 Anil K. Seth,University of Sussex,http://www.sussex.ac.uk/informatics/people/peoplelists/person/22981,3eJCZCkAAAAJ
 Anil Kumar Jain,Michigan State University,https://www.cse.msu.edu/~jain,g-_ZXGsAAAAJ
-Anil Kumar S. Vullikanti,Virginia Tech,https://www.bi.vt.edu/ndssl/people/Anil-Vullikanti,MNJ-E9UAAAAJ
+Anil Kumar S. Vullikanti,University of Virginia,https://www.bi.vt.edu/ndssl/people/Anil-Vullikanti,MNJ-E9UAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:916]: This URL is dead.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 2 above)
Comment (jiahao42) [reply csrankings.csv:916]: Fixed, thank you so much.

**Pull request merged**